config/localizations_native/ja/app.json

```diff
@@ -2,30 +2,30 @@
   "APP": {
     "GENERAL": {
       "TIME_STRINGS": {
-        "MINUTES": "",
-        "1MIN": "1分",
-        "5MIN": "5分",
+        "MINUTES": "分",
+        "1MIN": "1M",
```

Comment:
Inconsistency between 1MIN and 15MIN, etc...   Also inconsistency in other TIME_STRINGS.